locales/en.default.schema.json

```diff
@@ -1128,7 +1131,12 @@
           "name": "Price"
         },
         "quantity_selector": {
-          "name": "Quantity selector"
+          "name": "Quantity selector",
+          "settings": {
+            "quantity_limit": {
+              "label": "Maximum quantity"
```

Comment:
@melissaperreault is there another label for this setting? I made this up

Reply:
```suggestion
              "label": "Maximum quantity selector"
```
Yes, here it is!